services/discordService.js

```diff
@@ -64,8 +64,24 @@ const addRoleToUser = async (userid, roleid) => {
   return response;
 };
 
+const setDiscordNickname = async (userName, discordId) => {
+  const authToken = jwt.sign({}, config.get("rdsServerlessBot.rdsServerLessPrivateKey"), {
+    algorithm: "RS256",
+    expiresIn: config.get("rdsServerlessBot.ttl"),
+  });
+
+  await (
+    await fetch(`${DISCORD_BASE_URL}/guild/member`, {
+      method: "PATCH",
+      body: JSON.stringify({ userName, discordId }),
+      headers: { "Content-Type": "application/json", Authorization: `Bearer ${authToken}` },
+    })
+  ).json();
+};
+
```

Comment:
Nit: Want to Know this
Is this api is inbuild `${DISCORD_BASE_URL}/guild/member`
I didn't understand what this fetch is doing??

Can you give me some idea??